src/main/java/logic/Main.java

```diff
@@ -0,0 +1,4 @@
+package logic;
+
+public class Main {
```

Comment:
A) this is the wrong package, everything should be a subpackage of project
B) except the atual Main package (project.Main) nothing should be called Main